src/plugin/urlfilter-domaindenylist/plugin.xml

```diff
@@ -16,13 +16,13 @@
  limitations under the License.
 -->
 <plugin
-   id="urlfilter-domainblacklist"
-   name="Domain Blacklist URL Filter"
+   id="urlfilter-domaindenylist"
+   name="Domain Denylist URL Filter"
    version="1.0.0"
    provider-name="nutch.org">
 
    <runtime>
-      <library name="urlfilter-domainblacklist.jar">
+      <library name="urlfilter-denyblacklist.jar">
```

Comment:
Should be "urlfilter-domaindenylist.jar".

Reply:
Fixed this thanks....now git grep -F "domainblacklist" , just prints the CHANGES.txt since I included the changes to the plugin name and field name.